### DISCUSSION TITLE
Plugin GUI unresponsive until locally clicked on server system

### DISCUSSION THREAD

**Discussion by jamiekuse** [apohl79/audiogridder#404] [Help]
Hi there.

I have had this problem with both Beta8 and 1.1.0 on both windows and mac servers. Plugin running on Cubase/OSX.

The plugin GUI loads fine, but I cannot click or move any of the parameters on the plugin until I go on the server machine and click somewhere. Then it work as normal on the plugin.

Any tips? I figure it must be a problem on the plugin side since it has happened on two different fresh install servers on different OS.

**Comment by CaBuffalo** [comment]
Hey, I don't have any problem of what you described. I have tested AG in
the configuration of Logic as DAW and Windows 10 server; both Windows 10
server/client.

To me, it looks lilke you have something happened on your server side when
there is a plugin activated in AG server. It put the plugin as an inactive
window. Is there any common program you installed on both your MAC and
Windows server?

On Mon, Feb 22, 2021 at 10:56 AM jamiekuse <notifications@github.com> wrote:

> Hi there.
>
> I have had this problem with both Beta8 and 1.1.0 on both windows and mac
> servers. Plugin running on Cubase/OSX.
>
> The plugin GUI loads fine, but I cannot click or move any of the
> parameters on the plugin until I go on the server machine and click
> somewhere. Then it work as normal on the plugin.
>
> Any tips? I figure it must be a problem on the plugin side since it has
> happened on two different fresh install servers on different OS.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/404>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6A4KFES6GLI6ACECFTTAKSGFANCNFSM4YBB46PQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by jamiekuse** [reply]
There is not, and the windows machine I installed it on is a brand new windows install with essentially nothing other than AG and acoustica plugins.

I should also note that I have only experienced this with Acoustica plugins as I dont use AG for anything else

**Reply by apohl79** [reply]
Is this happening only after booting the server machine or every time you restart AG server or even every time you load a plugin?

If this happens only after booting, can you try to log into the server first before loading the first plugin?

**Comment by jamiekuse** [comment]
I know it happens after booting, because often it will happen again after
me manually clicking on the interface physically on the server machine. I
cannot reliably create the bug, it just happens every so often throughout
the day.

On Mon, Feb 22, 2021 at 1:08 PM Andreas Pohl <notifications@github.com>
wrote:

> Is this happen only after booting the server machine or every time you
> restart AG server or even every time you load a plugin?
>
> If this happens only after booting, can you try to log into the server
> first before loading the first plugin?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/404#discussioncomment-395147>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARSRIBJ5K5IDXVJGLI3KBSLTALBTFANCNFSM4YBB46PQ>
> .
>


-- 
Jamie Kuse
Engineer/Producer
www.jamiekuse.com
Co-Founder of NOVA Studios <http://novastudios.ca/>

**Reply by apohl79** [reply]
Does it also happen if you keep the UI session open?

Can you try to establish a VNC session in which you run AG server. You should be able to close the client afterwards as VNC keeps the UI session open.

**Comment by ALEXMCGINNESS** [comment]
i have the same issue. I have 3 servers. Sometimes 1 server will work where I can control the graphic interface from my main machine sometimes 2 will work properly but never three servers work from the main machine unless I go to the server and move a parameter than go back to the main machine and it will work.

**Reply by lesbrunn** [reply]
I've experienced something similar, though not as dramatic as what is being described here. After opening a plugin in the DAW, it doesn't respond for a couple of seconds, but then it does. It seems like it's a little laggy while the plugin connects with the server.

**Comment by fallynnknivez** [comment]
i know this is old, but i ran into the same issue yesterday with this, and wanted to share in case it can help future readers.

For me, this was happening with 1.2.0 beta, and it turned out that any time i was remoted in to the server pc (using teamviewer), for some reason the plugins where not taking focus, resulting in having to physically click the plugin window on the server pc, before commands would be picked up by the plugin's window.

To resolve the issue, i could either make sure i was not remoted into it (which wouldnt work for my purposes), so i went under the properties of audiogridder server and set it to run as administrator and the issue resolved. (This can potentially, cause issues with other apps getting focus, if your running other things on your server, that you need to also have focus at the same time. Just be aware)

Note: due to another issue, i went back to v1.1.1, and did not have to set it to run as admin to work correctly, tho, this may be due to having an extra click to bring up the GUI. Im not entirely sure.

**Comment by MingutaMing** [comment]
On Mac (Server Host) .. Go to System Preferences/Security & Privacy/Accessibility
-Unlock Make Changes 
-click AudioGridderServer.app on Allow the apps below to control your computer Window. 
Bingo.

**Reply by SINCd** [reply]
Hey thanks, but this did not work for me.  I still have to click randomly on a plugin, move a knob or pretend I can see a menu dropping open, click in mid-air, close the plugin and reopen it in hopes of seeing that I clicked on what I wanted or turned the knob in the right direction. I wish it was this simple. lol. Glad it's working for you though. I would love to know a working solution to this... I'm not running it from another computer, I'm running it from my Mac through crossover and using it in logic on venture.  I'm using the "crossover" version that was recommended in a tutorial to run crossgridder reliably.  Both audiogridder and crossover have permission in Accessibility. sad yas :(
